lib/marginalia/comment.rb

```diff
@@ -112,10 +112,12 @@ def self.sidekiq_job
       def self.line
         Marginalia::Comment.lines_to_ignore ||= DEFAULT_LINES_TO_IGNORE_REGEX
 
-        last_line = caller.detect do |line|
-          line !~ Marginalia::Comment.lines_to_ignore
+        last_line = caller_locations.detect do |loc|
```

Comment:
@fatkodima before I dive into it... did you ever look at using `Thread.each_caller_location` in this gem? I know you did in Rails but not sure if there was a reason to not do it here too.

Reply:
> did you ever look at using `Thread.each_caller_location` in this gem?

I think I don't.